src/lib.rs

```diff
@@ -10,6 +10,8 @@ use notify::{RecommendedWatcher, RecursiveMode, Watcher};
 use structopt::StructOpt;
 use wasm_bindgen_cli_support::Bindgen;
 
+pub use camino;
```

Comment:
1. Get camino from cargo_metadata
2. Remove camino from our dependencies

```suggestion
pub use cargo_metadata::camino;
```